src/views/Books/BookDetail.js

```diff
@@ -12,7 +14,12 @@ function BookDetail() {
 
   if (!book) return <h3>Loading book...</h3>;
 
-  return <Book book={book} showDetail />;
+  return (
+    <div>
+      <a href='/books'>Back to catalog</a>
```

Comment:
Remember to use Link here within react-router-dom to prevent a full page reload.